RetsTalk/RetsTalk/User/Controller/UserSettingViewController.swift

```diff
@@ -49,6 +46,29 @@ final class UserSettingViewController<T: UserSettingManageable>: UIHostingContro
     @objc private func backwardButtonTapped() {}
 }
 
+// MARK: - UserSettingManageableDelegate conformance
+
+extension UserSettingViewController: UserSettingManageableDelegate {
+    typealias Situation = UserSettingViewSituation
+
+    func alertNeedNotificationPermission(_ userSettingManageable: any UserSettingManageable) {
+        let alertAction = UIAlertAction(title: Texts.accept, style: .default) { _ in
+            if let url = URL(string: UIApplication.openSettingsURLString) {
+                UIApplication.shared.open(url)
+            }
+        }
+        let cancel = UIAlertAction(title: Texts.cancel, style: .cancel)
+        presentAlert(for: .needNotifactionPermission, actions: [alertAction, cancel])
+    }
```

Comment:
```suggestion
    func alertNeedNotificationPermission(_ userSettingManageable: any UserSettingManageable) {
        let acceptAction = UIAlertAction(title: Texts.accept, style: .default) { _ in
            if let url = URL(string: UIApplication.openSettingsURLString) {
                UIApplication.shared.open(url)
            }
        }
        let cancelAction = UIAlertAction(title: Texts.cancel, style: .cancel)
        presentAlert(for: .needNotifactionPermission, actions: [accpetAction, cancelAction])
    }
```
네이밍이 조금 모호해서 개선해봤습니다. accept가 아니더라도 적절한 네이밍으로 변경하면 좋을 것 같아요!